ExomeDepth/make_BEDdetail.py

```diff
@@ -158,8 +158,15 @@ def slice_vcf(args, merge_dic, lims):
                     deldupratio = float(record.genotype(sampleid)['PD'])
                     totalcalls = int(record.genotype(sampleid)['TC'])
                     refsamples = int(record.genotype(sampleid)['RS'])
-                    ratio = float(record.genotype(sampleid)['RT'])
-                    ccn = float(record.genotype(sampleid)['CCN'])
+
+                    if "SM" in record.FORMAT: # New Franklin format
+                        ratio = float(record.genotype(sampleid)['SM'])
+                    else:  # Old Alissa format
+                        ratio = float(record.genotype(sampleid)['RT'])
+                    if "CCN" not in record.FORMAT and "CN" in record.FORMAT: # New Franklin format
+                        ccn = float(record.genotype(sampleid)['CN'])
+                    else:   # Old Alissa format
+                        ccn = float(record.genotype(sampleid)['CCN'])
```

Comment:
Can you combine this:
```python
if "SM" in record.FORMAT and "CCN" not in record.FORMAT and "CN" in record.FORMAT:
   ratio = float(record.genotype(sampleid)['SM'])
   ccn = float(record.genotype(sampleid)['CN'])
else:
    ratio = float(record.genotype(sampleid)['RT'])
    ccn = float(record.genotype(sampleid)['CCN'])
```

If 'SM' and 'CN' combination is enough to check if it is a 'franklin' vcf you can also remove the 'not' CNN.